src/training/pango/pango_font_info.cpp

```diff
@@ -230,12 +230,20 @@ bool PangoFontInfo::CoversUTF8Text(const char *utf8_text, int byte_length) const
       int len = it.get_utf8(tmp);
       tmp[len] = '\0';
       tlog(2, "'%s' (U+%x) not covered by font\n", tmp, *it);
+#if PANGO_VERSION_CHECK(1, 52, 0)
```

Comment:
Thanks for looking this up. So the check could be modified to check for 1.44.0. Since 1.52.0 pango_coverage_unref is marked as deprecated and causes a compiler warning.